includes/Core/Tags/First_Party_Mode/First_Party_Mode.php

```diff
@@ -72,4 +73,23 @@ public function register() {
 		$this->first_party_mode_settings->register();
 		$this->rest_controller->register();
 	}
+
+	/**
+	 * Gets an array of debug field definitions.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	public function get_debug_fields() {
+		$settings = $this->first_party_mode_settings->get();
+
+		return array(
+			'first_party_mode_is_enabled' => array(
+				'label' => __( 'First Party Mode: Enabled', 'google-site-kit' ),
```

Comment:
@techanvil, should it be `First-Party Mode: Enabled` with a hyphen?

Reply:
Thanks for spotting that @hussain-t, you are quite right, "First-Party" should be hyphenated.

I've amended the AC/IB for this issue, and for https://github.com/google/site-kit-wp/issues/9668 accordingly.

Cc @benbowler

Reply:
Thanks @techanvil. Since I'm reviewing the PR I can amend the changes.

cc: @benbowler

Reply:
Sounds good, thanks Hussain!